### DISCUSSION TITLE
Pin mode: Allow any application to float in front of all windows

### DISCUSSION THREAD

**Discussion by maymaymaybe8** [rxhanson/Rectangle#1480] [General]
The current pin mode only allows you to select one app to appear in a specific location, is it possible to allow any app to be topped or appear in a specific location via rectangle pro hotkeys?

**Comment by rxhanson** [comment]
Unfortunately, this is a macOS limitation where Rectangle cannot make windows of other apps be "always on top". The closest it would be able to do is float a window that uses screen recording to display what's in another window. I don't have plans to add this to Rectangle at this time.

**Reply by maymaymaybe8** [reply]
Ok, got it.

**Reply by CemKutlar** [reply]
There is an app that is doing this but it is paid. I believe there should be a way to do this. I can share the app name if it is not considered an ad.

**Reply by maymaymaybe8** [reply]
> There is an app that is doing this but it is paid. I believe there should be a way to do this. I can share the app name if it is not considered an ad.

Thank you. Are you referring to BetterTouchTool? I bought this app as well, but I rarely use it except for this function. That’s why I was wondering if Rectangle could add this feature, so I could reduce the number of apps I use. If you’re not referring to BetterTouchTool, I would like to know the name of the app. Anyway, thanks for your kindness.

**Reply by CemKutlar** [reply]
Yes I refer that app. I think it is somehow achievable.

**Reply by rxhanson** [reply]
Maybe try this one? https://github.com/williamcpierce/Overview

**Reply by maymaymaybe8** [reply]
> Maybe try this one? https://github.com/williamcpierce/Overview

Thanks, I'll try this app.
